### PR TITLE
Version gate node version

### PR DESCRIPTION
**Pull request by ajacksified**
:eyeglasses: @umbrae

### CODE REVIEW

File: version.js [hunk 1]
```diff
@@ -0,0 +1,6 @@
+var semver = require('semver');
+var packageJSON = require('./package.json');
+
+if (!semver.satisfies(process.version, packageJSON.engines.node)) {
```
Comment (umbrae) [inline version.js]: This will make it bomb out on iojs, right? If so, is there a reason we need `iojs` in `package.json`'s `engines`?

**Comment by umbrae** [comment]
:nail_care: just a small q!

**Comment by ajacksified** [comment]
Oh, derp

**Comment by ajacksified** [comment]
:haircut:

**Comment by umbrae** [comment]
:fish:

**Pull request merged**